src/logic/encodable/basic.lean

```diff
@@ -53,6 +54,10 @@ theorem encode_injective [encodable α] : function.injective (@encode α _)
 @[simp] lemma encode_inj [encodable α] {a b : α} : encode a = encode b ↔ a = b :=
 encode_injective.eq_iff
 
+-- The priority of the instance below is less than the priorities of `subtype.countable`
+-- and `quotient.countable`
```

Comment:
Can you elaborate on why this is so?

Otherwise LGTM

bors d+

Reply:
I'll ask on Zulip.